python/ray/tests/test_basic.py

```diff
@@ -1236,13 +1237,17 @@ def __init__(self):
         def method(self):
             pass
 
-    a = Actor._remote(
-        args=[0], kwargs={"y": 1}, num_gpus=1, resources={"Custom": 1})
+    a = Actor.options(
+        kwargs={
+            "y": 1
+        }, num_gpus=1, resources={
+            "Custom": 1
+        }).remote(args=[0])
 
-    a2 = Actor2._remote()
-    ray.get(a2.method._remote())
+    a2 = Actor2.remote()
+    ray.get(a2.method.remote())
 
-    id1, id2, id3, id4 = a.method._remote(
+    id1, id2, id3, id4 = a.method.remote(
```

Comment:
need to change this to use `.options().remote()`

Reply:
Here `a` is Actor and the ActorMethod class which `a.method` is doesn't even have the .options function.